processor/batchprocessor/config.go

```diff
@@ -42,5 +53,13 @@ func (cfg *Config) Validate() error {
 	if cfg.SendBatchMaxSize > 0 && cfg.SendBatchMaxSize < cfg.SendBatchSize {
 		return errors.New("send_batch_max_size must be greater or equal to send_batch_size")
 	}
+	uniq := map[string]bool{}
+	for _, k := range cfg.MetadataKeys {
+		l := strings.ToLower(k)
+		if _, has := uniq[l]; has {
+			return errors.New("duplicate entry in metadata_keys")
```

Comment:
Not a big deal, but perhaps it would be nice to point to the duplicate key in the log message.